modules/caddyhttp/requestbody/requestbody.go

```diff
@@ -31,6 +32,7 @@ type RequestBody struct {
 	// The maximum number of bytes to allow reading from the body by a later handler.
 	// If more bytes are read, an error with HTTP status 413 is returned.
 	MaxSize int64 `json:"max_size,omitempty"`
+	Replace string `json:"replace,omitempty"`
```

Comment:
Would need a helpful godoc comment on this field.

Reply:
Actually, can we rename this to `Set`? I keep thinking of "replacing content within the body" whereas "setting the body" is more precise IMO.